hydra_base/util/hdb.py

```diff
@@ -107,9 +107,9 @@ def make_root_user():
 
 
     db.DBSession.flush()
-    
```

Comment:
This file has no changes. Don't include in merge request please

Reply:
Atom deleted the tabs/spaces on empty lines and then finished into the diffs